brownie_hooks.py

```diff
@@ -0,0 +1,3 @@
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
How are you using this file? are you calling 
`python brownie_hooks.py` before working on the repo?

Reply:
It's automatically used by brownie and makes it so that it picks up the .env file

Reply:
If you end up using `export`, the ones you exported end up taking precedence, making it a optional use (i.e. you can continue using your old setup with no changes)

Reply:
TIL

https://github.com/eth-brownie/brownie/blob/f4ba974488b67d1f5ad4acfffc8603af912c31d4/brownie/project/main.py#L946-L949